sw/otbn/crypto/p256_ecdsa_sca.s

```diff
@@ -33,33 +33,66 @@ p256_ecdsa_verify:
   jal      x1, p256_verify
   ecall
 
-.bss
+.data
 
 /* Freely available DMEM space. */
 
 /* Operation mode (1 = sign; 2 = verify) */
 .globl mode
 .balign 4
 mode:
-  .zero 4
+  .word 0x1
 
 /* All constants below must be 256b-aligned. */
 
 /* random scalar k (in two shares) */
 .global k0
 .balign 32
 k0:
-  .zero 32
+  /* Word-order: The word with the lowest index is last */
+  /* Byte order in a word: little-endian */
+  /* i.e., k0 = 0x0000000...ffffffff. */
+  .word 0xffffffff
+  .word 0xffffffff
+  .word 0xffffffff
+  .word 0xffffffff
+  .word 0x00000000
+  .word 0x00000000
+  .word 0x00000000
+  .word 0x00000000
+
 .global k1
 .balign 32
 k1:
-  .zero 32
+  /* Word-order: The word with the lowest index is last */
+  /* Byte order in a word: little-endian */
+  /* i.e., k1= 0x0000000...ffffffff. */
+  .word 0x00000000
+  .word 0x00000000
+  .word 0x00000000
+  .word 0x00000000
+  .word 0x00000000
+  .word 0x00000000
+  .word 0x00000000
+  .word 0x00000000
+
 
 /* message digest */
 .globl msg
 .balign 32
 msg:
-  .zero 32
+  /*"Hello OTBN."*/
+  /* Word-order: The word with the lowest index is last */
```

Comment:
I was surprised when I first read this but I think the comment (all occurences) needs to be fixed:
```suggestion
  /* Word-order: The word with the lowest index is first */
```

Reply:
Thanks, the comment was misleading and redundant. I removed that comment. Instead, I just keep the value I intend to set. For example, `/* msg = 0x06d71207...4456fd21 */`.